README.md

```diff
@@ -1 +1,73 @@
-# MovieBadgers
\ No newline at end of file
+# Movie Badgers by [HackBadgers](https://github.com/HackBadgers)
+
+## Description
+Movie Badgers is a [Flutter](https://flutter.dev/) project of a movie REST API consumer application. With this project, we intend to demonstrate concepts of good practices, SOLID, Clean Architecture, TDD using agile development process - AGILE and continuous integration and delivery.
+
+## Installation
+
+```dart
+val under = "construction"
+```
+
+## Usage
+
+```dart
+val under = "construction"
+```
+
+## Contributing
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
+
+### Naming branches and tasks
+  1. Create an User Story [id](https://www.uuidgenerator.net/version4), e.g. 0a715730
+  2. Create an User Story card on project board.
+  ```bash
+My User Story Name - 0a715730
+```
+  3. Create an Task [id](https://www.uuidgenerator.net/version4), e.g. b2b33e12
+  4. Create an Task card on project board.
+  ```bash
+b2b33e12 - My Task Name
+```
+  5. Create a git branch using US and Task Ids to start coding.
+```bash
+git branch -a us/0a715730/task/b2b33e12
+```
+
+### How to write a good commit
+  Follow udacity good practices.
+  Follow [Udacity]() good practices.
+
+  ```bash
+type: subject
+
+body
+
+footer
+```
+The title consists of the type of the message and subject.
+
+#### The Type
+
+The type is contained within the title and can be one of these types:
+
+* feat: a new feature
```

Comment:
change only to the full name, already have commits in the project by the full name

`* feature: a new feature`